monai/transforms/composables.py

```diff
@@ -360,65 +383,40 @@ def __call__(self, data):
 
 
 class RandGaussianNoised(Randomizable, MapTransform):
-    """Add gaussian noise to image.
+    """Add gaussian noise to image. Suppose all the expected fields have same shape.
```

Comment:
'gaussian' -> 'Gaussian', 'Suppose' -> 'This transform assumes'

Reply:
changed in the latest patch.
Thanks.